agent/src/Handler/Apps.hs

```diff
@@ -350,12 +353,17 @@ postUninstallAppLogic appId dryrun = do
         Just CreatingBackup  -> throwError (TemporarilyForbiddenE appId "uninstall" (show CreatingBackup))
         Just RestoringBackup -> throwError (TemporarilyForbiddenE appId "uninstall" (show RestoringBackup))
         _                    -> pure ()
-    let flags = if coerce dryrun then Left dryrun else Right (AppMgr2.Purge True)
-    breakageIds <- HM.keys . AppMgr2.unBreakageMap <$> AppMgr2.remove flags appId
-    bs <- pure (traverse (hydrate $ (AppMgr2.infoResTitle &&& AppMgr2.infoResVersion) <$> serverApps) breakageIds)
-        `orThrowM` InternalE "Reported app breakage for app that isn't installed, contact support"
+    let flags = if coerce dryrun then Left dryrun else Right (AppMgr2.Purge True, revert)
```

Comment:
Need to support both dryrun + revert flags to give users a heads up before they sign off on revert

Reply:
I addressed this now.